qa/tasks/cephfs/test_volumes.py

```diff
@@ -1143,6 +1151,51 @@ def test_subvolume_snapshot_create_and_rm(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_snapshot_info(self):
+
+        # tests the 'fs subvolume snapshot info' command
```

Comment:
nit: There was a suggestion from Patrick sometime ago to follow [this](https://github.com/ceph/ceph/blob/e6d0c18c24eece567d6ff5c119a229a48b179c40/qa/tasks/cephfs/test_volumes.py#L520) type of comments for the function headers.

Reply:
Done